pdr-admin/src/app/protected-area/protected-area-manage/protected-area-edit-form/protected-area-edit-form.component.ts

```diff
@@ -136,12 +147,34 @@ export class ProtectedAreaEditFormComponent {
 
     let mergedObj = { ...baseObj, ...this.modalObj };
 
+    // We want display name to be same as legal name
+    // API will set this if displayName is null
+    if (this.hideDisplayName) {
+      mergedObj.displayName = null;
+    }
+
     await this.protectedAreaService.edit(this.currentData.pk, mergedObj, this.updateType);
     this.protectedAreaService.fetchData(this.currentData.pk);
     this.confirmSaveClose.nativeElement.click();
     this.router.navigate(['protected-areas', this.currentData.pk]);
   }
 
+  toggleDisplayNameForm() {
+    if (this.hideDisplayName) {
+      // Toggle off
+      // Remove form control
+      this.form.removeControl('displayName');
+    } else {
+      // Toggle on
+      this.form.addControl(
+        'displayName',
+        new UntypedFormControl(null, { nonNullable: true, validators: [Validators.required] })
```

Comment:
Consider: when toggled on, default value should be either:

- existing record's `displayName` value, or if it is doesn't exist,
- existing record's `legalName` value